src/Features/Core/Portable/EmbeddedLanguages/Json/LanguageServices/JsonClassifier.cs

```diff
@@ -17,27 +21,33 @@ namespace Microsoft.CodeAnalysis.Features.EmbeddedLanguages.Json.LanguageService
     /// <summary>
     /// Classifier impl for embedded json strings.
     /// </summary>
-    internal abstract class AbstractJsonClassifier : IEmbeddedLanguageClassifier
+    [ExportEmbeddedLanguageClassifier(
+        PredefinedEmbeddedLanguageNames.Json,
+        new[] { LanguageNames.CSharp, LanguageNames.VisualBasic },
```

Comment:
would it be worth defining this array somewhere so we can just pass it in similar to `PredefinedEmbeddedLanguageNames.Json`?  Maybe one for just C# too

Reply:
Tears :'(